blog/2025/clio-2.5.0.md

```diff
@@ -0,0 +1,258 @@
+- Add ability for nodes to share information or status via the database ([#1976](https://github.com/XRPLF/clio/pull/1976)) by [@kuznetsss](https://github.com/kuznetsss)
+- Update forwarding cache to queue similar requests during entry updates ([#1997](https://github.com/XRPLF/clio/pull/1997)) by [@kuznetsss](https://github.com/kuznetsss)
+- Introduce `release_impl.yml` workflow to automatically create releases ([#2043](https://github.com/XRPLF/clio/pull/2043)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Implement clang build analyzer ([#2072](https://github.com/XRPLF/clio/pull/2072)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Add load balancer prometheus metrics ([#2096](https://github.com/XRPLF/clio/pull/2096)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Implement `FetchLedgerBySeq` caching to address uneven DB shard access ([#2014](https://github.com/XRPLF/clio/pull/2014)) by [@PeterChen13579](https://github.com/PeterChen13579)
+- Add support for Dosguard API weights ([#2082](https://github.com/XRPLF/clio/pull/2082)) by [@kuznetsss](https://github.com/kuznetsss)
+- Implement load balancer metrics in ETLng ([#2119](https://github.com/XRPLF/clio/pull/2119)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Add hadolint Docker pre-commit hook ([#2135](https://github.com/XRPLF/clio/pull/2135)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Add account permission support ([#2145](https://github.com/XRPLF/clio/pull/2145)) by [@kuznetsss](https://github.com/kuznetsss)
+- Add get range functionality to export tool ([#2131](https://github.com/XRPLF/clio/pull/2131)) by [@PeterChen13579](https://github.com/PeterChen13579)
+- Refactor ETLng publisher and service ([#2138](https://github.com/XRPLF/clio/pull/2138)) by [@godexsoft](https://github.com/godexsoft)
+- Add ETLng MPT support ([#2154](https://github.com/XRPLF/clio/pull/2154)) by [@godexsoft](https://github.com/godexsoft)
+- Add support for Batch transactions ([#2162](https://github.com/XRPLF/clio/pull/2162)) by [@PeterChen13579](https://github.com/PeterChen13579)
+- Update Clio to use the new web server by default ([#2182](https://github.com/XRPLF/clio/pull/2182)) by [@kuznetsss](https://github.com/kuznetsss)
+- Add ability to create releases in CI ([#2168](https://github.com/XRPLF/clio/pull/2168)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Add new Docker images structure, tools image ([#2185](https://github.com/XRPLF/clio/pull/2185)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Add ubsan build to nightly release ([#2190](https://github.com/XRPLF/clio/pull/2190)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Add support for Permissioned DEX ([#2152](https://github.com/XRPLF/clio/pull/2152)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Switch Clio to use Conan v2 ([#2179](https://github.com/XRPLF/clio/pull/2179)) by [@mathbunnyru](https://github.com/mathbunnyru)
```

Comment:
Improvements to the build system like this also seem like they should be categorized elsewhere too.